CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 
 cmake_minimum_required (VERSION 3.1)
 
-set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_CXX_STANDARD 14)
```

Comment:
drive-by comment: You should probably set `CMAKE_CXX_STANDARD_REQUIRED` to `true` also.

https://cmake.org/cmake/help/v3.7/variable/CMAKE_CXX_STANDARD_REQUIRED.html